### PR TITLE
Add classification plots

### PR DESCRIPTION
**Pull request by jonathan-paugh**
Adds capability to generate classification plots based unique values in the label. Should be used with binary-labelled data to get meaningful results.

Adds a data visualization menu to generate specific plots.

### CODE REVIEW

File: core/data_visualization.py [hunk 1]
```diff
@@ -53,25 +80,46 @@ def plot_hist(x, bins, x_axis, y_axis, file_name):
...
-    generate_compound_plots(features, label)
+        classes = _get_classes(x, y, label)
+        plot_classification(
+            classes.keys(),
+            [
+                [x for x, _ in c] for c in list(classes.values())
+            ],
+            [
+                [y for _, y in c] for c in list(classes.values())
+            ],
```
Comment (semibran) [inline core/data_visualization.py]: black magic version
```suggestion
            *zip(*classes.values()),
```

**Review by semibran — APPROVED** [review]

File: core/data_visualization.py (hunk 1 above)
Comment (jonathan-paugh) [reply core/data_visualization.py]: Doesn't seem to work when I test it.

Comment (semibran) [reply core/data_visualization.py]: God I misread it's cause it's supposed to be even worse than I thought
```py
*zip(*(zip(*points) for points in classes.values())),
```

Comment (jonathan-paugh) [reply core/data_visualization.py]: Oh god, I love it!

**Pull request merged**